package.json

```diff
@@ -145,6 +145,11 @@
             "default": false,
             "markdownDescription": "**Experimental**: Run `swift build` in the background whenever a file is saved.",
             "order": 7
+          },
+          "swift.buildPath": {
+            "type": "string",
```

Comment:
We need to add a description for the setting. Also if you could add a more detailed description in `docs/settings.md`.